test_util/test_aws_cf.py

```diff
@@ -0,0 +1,140 @@
+#!/usr/bin/env python3
+"""Deploys DCOS AWS CF template and then runs integration_test.py
+
+The following environment variables control test procedure:
+
+DCOS_TEMPLATE_URL: string
+    The template to be used for deployment testing
+
+CI_FLAGS: string (default=None)
+    If provided, this string will be passed directly to py.test as in:
+    py.test -vv CI_FLAGS integration_test.py
+
+TEST_ADD_ENV_*: string (default=None)
+    Any number of environment variables can be passed to integration_test.py if
+    prefixed with 'TEST_ADD_ENV_'. The prefix will be removed before passing
+"""
+import logging
+import os
+import random
+import stat
+import string
+import sys
+from contextlib import closing
+
+import test_util.provide_aws
+import test_util.test_runner
+from ssh.ssh_tunnel import SSHTunnel
+
+LOGGING_FORMAT = '[%(asctime)s|%(name)s|%(levelname)s]: %(message)s'
+logging.basicConfig(format=LOGGING_FORMAT, level=logging.DEBUG)
+log = logging.getLogger(__name__)
+
+DEFAULT_AWS_REGION = 'us-west-2'
```

Comment:
Need to keep doing the environment variable fetching like the other script does for getting this that @alberts added the other day.